src/App.js

```diff
@@ -1,41 +1,14 @@
 import "./App.css";
-import logo from "./logo.png";
+import { SideBar, TopBar } from "./components";
+import { Authenticate, LandingPage } from "./screens";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+    <div className={`${!true ? "dark" : ""} App`}>
+      {/* <Authenticate /> */}
+      {/* <LandingPage /> */}
+      <TopBar />
+      {/* <SideBar /> */}
```

Comment:
These comments can be removed

Reply:
Yup, these are for future reference when I will do the functionalities.